js/views/pauseScreen.js

```diff
@@ -246,18 +241,22 @@ class PauseScreen extends React.Component {
       'oo-hidden': !this.props.skinConfig.pauseScreen.showPauseIcon || this.props.pauseAnimationDisabled
     });
 
-    var titleMetadata = (
+    const titleMetadata = (
       <div className={titleClass} style={titleStyle}>
         {this.props.contentTree.title}
       </div>
     );
-    var descriptionMetadata = (
-      <div className={descriptionClass} ref="description" style={descriptionStyle}>
+    const descriptionMetadata = (
+      <div
+        className={descriptionClass}
+        ref={text => this.description = text}
```

Comment:
Even simple logic inside templates is a kind of antipattern.

Reply:
it would be better to save function for 'ref' now.

Reply:
I think this is an improvement over the deprecated string ref that we had before. In the future we should probably just use `createRef()` and then declare the ref such as `<div ref={this.description} >`

Reply:
I tried to use it, but had a problem (this.description was returned as undefined), and I decided to check it in future because it did not connect with the task.